src/wpool_pool.erl

```diff
@@ -405,10 +405,15 @@ min_message_queue(Size, #wpool{size = Size}, Found) ->
   Worker;
 min_message_queue(Checked, Wpool, Found) ->
   Worker = worker_name(Wpool#wpool.name, Wpool#wpool.next),
-  case erlang:process_info(erlang:whereis(Worker), message_queue_len) of
-    {message_queue_len, 0} -> Worker;
-    {message_queue_len, L} ->
-      min_message_queue(Checked + 1, next_wpool(Wpool), [{L, Worker} | Found])
+  QLength = queue_length(whereis(Worker)),
+  min_message_queue(Checked + 1, next_wpool(Wpool), [{QLength, Worker} | Found]).
```

Comment:
Line 409 is too long:   min_message_queue(Checked + 1, next_wpool(Wpool), [{QLength, Worker} | Found])..